src/content/en/fundamentals/security/_toc.yaml

```diff
@@ -8,8 +8,9 @@ toc:
       path: /web/fundamentals/security/csp/
     - title: Encrypting Data In Transit
       section:
-      - title: Why HTTPS
+      - title: Why HTTPS matters
         path: /web/fundamentals/security/encrypt-in-transit/why-https
```

Comment:
can you set the URL here to the web.dev URL so it doesn't have to go through the redirect?